mapper.go

```diff
@@ -58,7 +59,7 @@ func (m *metricMapper) initFromString(fileContents string) error {
 
 		switch state {
 		case SEARCHING:
-			if line == "" {
+			if line == "" || line[0] == COMMENT_CHAR {
```

Comment:
You'd be required to keep a blank line between .

@juliusv What do you think? This will end up in the other two exporters doing mappings too.